docs/pages/modules/native-view-tutorial.mdx

```diff
@@ -474,7 +474,7 @@ export default function App() {
 
 ## Bonus: Build a web browser UI around it
```

Comment:
In this section, please update the screenshot order to Android and iOS and use `ContentSpotlight` component to display the static image.